CONTRIBUTING.md

```diff
@@ -5,9 +5,9 @@ The search-gov repository includes the work of multiple contributors. You're enc
 #### Contributing to Search.gov Multilingual Search Support
 Note: As of November 2021, management of multilingual support has moved from the [GSA/Punchcard](https://github.com/GSA/punchcard/) repository to the [GSA/search-gov](https://github.com/GSA/search-gov) repository.
 
-The search-gov application uses the YAML files in [config/locales](https://github.com/GSA/search-gov/tree/master/config/locales) to provide localized translations of text strings based on the locale set for the user.
+The search-gov application uses the YAML files in [config/locales](https://github.com/GSA/search-gov/tree/main/config/locales) to provide localized translations of text strings based on the locale set for the user.
```

Comment:
As long as we're updating these, let's follow best practices and use relative links to files within the repo:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links